### PR TITLE
[bn] localization update `content/bn/_index.md`

### PR DESCRIPTION
**Pull request by Zubiii333**
…cal errors

### Describe your changes
Used simplified bangla translation and fixed a few punctuational mistakes

### Related issue number or link (ex: `resolves #issue-number`)

Related Issue: #410 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/bn/_index.md [hunk 1]
```diff
@@ -10,6 +10,33 @@ status: Completed
 ## অবদান
 
 ক্লাউড নেটিভ শব্দকোষে পরিবর্তন, সংযোজন এবং উন্নতির পরামর্শ দেওয়ার জন্য সবাইকে আমন্ত্রণ জানানো হয়েছে। আমরা এই ভাগ করা অভিধানের বিকাশ এবং উন্নতির জন্য CNCF দ্বারা পরিচালিত একটি সম্প্রদায়-চালিত প্রক্রিয়া নিযুক্ত করি। এই শব্দকোষটি ক্লাউড নেটিভ প্রযুক্তির আশেপাশে একটি ভাগ করা শব্দভাণ্ডার সংগঠিত করার জন্য একটি বিক্রেতা-নিরপেক্ষ প্ল্যাটফর্ম প্রদান করে৷ প্রজেক্টের উদ্দেশ্য এবং চার্টার মেনে চলা সকল অংশগ্রহণকারীদের থেকে অবদানকে স্বাগত জানানো হয়।
+---
+title: "ক্লাউড নেটিভ শব্দকোষ"
+status: Completed
+---
```
Comment (sajibAdhi) [inline content/bn/_index.md:16]: Hello @Zubiii333, those lines created an unexpected UI. Those should be at the start of the file at line 1.

image

**Review by sajibAdhi — CHANGES_REQUESTED** [review]
Hey @Zubiii333, There are duplications of sections. Kindly update those.

image

File: content/bn/_index.md (hunk 1 above)
Comment (Zubiii333) [reply content/bn/_index.md]: Hi @sajibAdhi ! Sorry for the issue created there. I believe its fixed now.

**Comment by sajibAdhi** [comment]
Hello @Zubiii333,

There are repetitions of sections such as `স্বীকৃতি` and `লাইসেন্স`. Kindly fix those.

image

**Comment by Zubiii333** [comment]
> 
Hi! I think its fixed now

**Review by sajibAdhi — APPROVED** [review]

**Review by sajibAdhi — APPROVED** [review]
LGTM

**Review by mitul3737 — APPROVED** [review]

**Pull request merged**